.pre-commit-config.yaml

```diff
@@ -1,56 +1,43 @@
+# All available hooks: https://pre-commit.com/hooks.html
+# R specific hooks: https://github.com/lorenzwalthert/precommit
 repos:
-#####
-# Basic file cleanliness
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.5.0
-    hooks:
-    -   id: check-added-large-files
-    -   id: check-yaml
-    -   id: check-toml
-    -   id: end-of-file-fixer
-    -   id: mixed-line-ending
-    -   id: trailing-whitespace
-#####
-# Python
--   repo: https://github.com/psf/black
-    rev: 23.10.0
-    hooks:
-        # if you have ipython notebooks, consider using
-        # `black-jupyter` hook instead
-    -   id: black
-        args: ['--line-length', '79']
--   repo: https://github.com/PyCQA/isort
-    rev: 5.12.0
-    hooks:
-    -   id: isort
-        args: ['--profile', 'black',
-               '--line-length', '79']
-- repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.1.0
-  hooks:
-    - id: ruff
-#####
 # R
 -   repo: https://github.com/lorenzwalthert/precommit
     rev: v0.4.2
     hooks:
     -   id: style-files
+        args: [--style_pkg=styler, --style_fun=tidyverse_style]    
```

Comment:
Are there any dev practices you want to recommend to avoid linter / styler conflicts? If so, let's put them in a readme in a later PR.

Reply:
I'm hopeful we're going to keep this simple enough to avoid cyclocompocalypse without explicit guidance. The hooks should be enough to get us started, but let's revisit if there are issues down the road?